package.json

```diff
@@ -31,24 +31,24 @@
     "trailingComma": "es5"
   },
   "devDependencies": {
-    "@types/jest": "^24.0.23",
-    "@types/react-redux": "^7.1.5",
-    "@types/redux-mock-store": "^1.0.1",
-    "@typescript-eslint/eslint-plugin": "^2.10.0",
-    "@typescript-eslint/parser": "^2.10.0",
+    "@types/jest": "^24.9.1",
```

Comment:
While I was in here, I updated this library's (i.e. build-time) dependencies `npm outdated` + `npm update`. All but one are just build-time dependencies only used for compilation, type-checking or running tests.